src/main/java/org/cryptomator/ui/recoverykey/RecoveryKeyComponent.java

```diff
@@ -38,11 +43,19 @@ default void showRecoveryKeyRecoverWindow() {
 		stage.show();
 	}
 
+	default void showOnboardingDialogWindow() {
+		Stage stage = window();
+		stage.setScene(recoverOnboardingScene().get());
+		stage.sizeToScene();
+		stage.show();
+	}
 
 	@Subcomponent.Factory
 	interface Factory {
 
-		RecoveryKeyComponent create(@BindsInstance @RecoveryKeyWindow Vault vault, @BindsInstance @Named("keyRecoveryOwner") Stage owner);
+		RecoveryKeyComponent create(@BindsInstance @RecoveryKeyWindow Vault vault, //
+									@BindsInstance @Named("keyRecoveryOwner") Stage owner, //
+									@BindsInstance @Named("recoverType") ObjectProperty<RecoveryActionType> recoverType);
```

Comment:
As mentioned above, do we really need to wrap the recoverType inside a property? It seems like we are only calling the `get()` method. If that is the case, the wrapping is not needed.